blocks/columns/columns.js

```diff
@@ -15,25 +14,29 @@ export default function decorate(block) {
       }
     });
   });
+  columnCollectionsBlock(block);
```

Comment:
I would move the `if` up here, before calling the function.

Reply:
done